disvae/models/losses.py

```diff
@@ -423,10 +423,9 @@ def __call__(self, data, recon_batch, latent_dist, is_train, storer, latent_samp
             storer['tc_loss'].append(tc_loss.item())
             storer['dw_kl_loss'].append(dw_kl_loss.item())
 
-            # TODO Remove this when visualisation fixed
-            tc_loss_vec = (logqz - logqz_prodmarginals)
+            dw_kl_loss_vec = (logqz_prodmarginals - logpz)
             for i in range(latent_dist.size(1)):
-                storer['kl_loss_' + str(i)].append(tc_loss_vec[i].item())
+                storer['dw_kl_loss_' + str(i)].append(dw_kl_loss_vec[i].item())
```

Comment:
@alecokas  - please check that this name change doesn't affect visualisation. I seem to recall that you made it name agnostic.

Reply:
`kl_loss_i` is dimension-wise, so I think keeping it with the original name is the right thing to do. That being said, the viz should still work

Reply:
Changed